src/nunjucks/moduk/components/warning-text/__examples__/with-html.njk

```diff
@@ -0,0 +1,10 @@
+{%- from "moduk/components/warning-text/macro.njk" import modukWarningText -%}
+
+{% set html %}
+  This operation cannot be undone. Would you like to proceed?<br/><a class="govuk-notification-banner__link" href="#">Go back</a>
```

Comment:
Maybe some other HTML would make sense as it might be a bit unusual to put a back link in there.